src/tsconfig-loader.ts

```diff
@@ -95,7 +95,12 @@ export function walkForTsConfig(
   directory: string,
   existsSync: (path: string) => boolean = fs.existsSync
 ): string | undefined {
-  const configPath = path.join(directory, "./tsconfig.json");
+  let configPath = path.join(directory, "./tsconfig.json");
+  if (existsSync(configPath)) {
+    return configPath;
+  }
+
+  configPath = path.join(directory, "./jsconfig.json");
```

Comment:
You could simplify this to

```suggestion
  if (existsSync(configPath)) {
    configPath = path.join(directory, "./jsconfig.json");
  }
```

Reply:
I don't think this will work. if `existsSync('tsconfig.json')`, we should return `tsconfig.json` path, right?

Reply:
Oh my mistake, the condition should be negated 🙈

Updated the suggestion

Reply:
commited. thanks for your suggestion